server/tests/db_test.rs

```diff
@@ -1,96 +1,98 @@
+//     let game = game_repo.get_game(game_id).await?;
+//     println!("{:?}", game);
+
+//     let participation_id = participation_repo
+//         .create_participation(game_id, user_id)
+//         .await?;
+//     let participation = participation_repo
+//         .get_participation(participation_id)
+//         .await?;
+//     println!("{:?}", participation);
+
+//     let won_game = game_repo.update_game_winner(game_id, user_id).await?;
+//     println!("{:?}", won_game);
+//     let user = user_repo
+//         .update_user(user_id, Some("new name"), None, Some("password"))
+//         .await?;
+//     println!("{:?}", user);
+
+//     participation_repo
+//         .delete_participation(participation_id)
+//         .await?;
+//     game_repo.delete_game(game_id).await?;
+//     // user_repo.delete_user(user_id).await?;
+
+//     let mut game = Game::new();
+
+//     println!("{:?}", game.connect_player(user_id).await);
+//     println!("{:?}", game.connect_player(99999).await);
+//     println!("{:?}", game.disconnect_player(user_id));
+//     println!("{:?}", game.connect_player(user_id).await);
+//     println!("{:?}", game.connect_player(user_id).await);
+//     println!("{:?}", game.connect_player(user2_id).await);
+
+//     println!("{:?}", game.player_flip_ready(user_id).await);
+//     println!("{:?}", game.player_flip_ready(user2_id).await);
+
+//     println!("{:?}", game.make_action(99999, 0, 1).await);
+//     println!("{:?}", game.make_action(user2_id, 0, 1).await);
+//     println!("{:?}", game.make_action(user_id, 0, 10).await);
+//     println!("{:?}", game.make_action(user_id, 20, 0).await);
+
+//     for _ in 0..6 {
+//         print_messages(game.make_action(user_id, 0, 0).await);
+//         print_messages(game.make_action(user2_id, 0, 0).await);
+//         print_messages(game.make_action(user_id, 0, 0).await);
+//         print_messages(game.make_action(user2_id, 0, 0).await);
+//         print_messages(game.make_action(user_id, 0, 0).await);
+//         print_messages(game.make_action(user2_id, 0, 0).await);
+//         print_messages(game.make_action(user_id, 0, 0).await);
+//         print_messages(game.make_action(user2_id, 0, 0).await);
+
+//         print_messages(game.make_action(user_id, 1, 0).await);
+//         print_messages(game.make_action(user2_id, 1, 0).await);
+//         print_messages(game.make_action(user_id, 1, 0).await);
+//         print_messages(game.make_action(user2_id, 1, 0).await);
+//         print_messages(game.make_action(user_id, 1, 0).await);
+//         print_messages(game.make_action(user2_id, 1, 0).await);
+//         print_messages(game.make_action(user_id, 1, 0).await);
+//         print_messages(game.make_action(user2_id, 1, 0).await);
+//     }
+
+//     Ok(())
+// }
+
+// fn print_messages(messages: Vec<ServerMessage>) {
+//     println!("---");
+//     messages
+//         .iter()
+//         .for_each(|message| println!("{:?}", message))
+// }
```

Comment:
Is there a reason to have a file whose content is fully commented out?

Reply:
Well, I'm not sure. This test fails since there is no DB set in gh action. But if we found way to add it (or have some permanent db elsewhere), we could use this test, so I did not want to remove it completely. On the other hand it is not that much code, so we could write it again later. Up to you, I guess.

Reply:
I see; in that case, I think it's okay. I did not realize that it was because of the pipeline.